### DISCUSSION TITLE
How to not create lightning_logs when using a external logger like wandb ?

### DISCUSSION THREAD

**Discussion by richarddwang** [Lightning-AI/pytorch-lightning#6685] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I would like my wandb logger to just place their data under `wandb` dir, and checkpointcallback to save ckpts under `dir_path` I specified.
And I don't want pl to create `lightning_logs` and files under it, but I can't set `logger=False` b/c I use a logger. Is there any suggestion ?

**Comment by awaelchli** [comment, marked as answer]
You can set the `save_dir` in WandbLogger, something like

```python
logger = WandbLogger(save_dir="wandb", ...)
Trainer(logger=logger, ...)
```
This should work (haven't tested it).
Then your logs and checkpoints will save to two different locations.

**Reply by richarddwang** [reply]
Hi, I found `WandbLogger`'s `save_dir` defaults to `wandb`. 
I accidentally forgot to delete `lightning_logs` and thought it was generated by the current run with wandb. Actually we just need to specify `WandbLogger` as logger, `dirpath` of `ModelCheckpoint`, and then `lightning_logs` won't be generated.
Anyway, thanks for your reply !
